packages/mobile/src/components/dropdown/dropdown.tsx

```diff
@@ -64,11 +71,18 @@ const DropDown = <T extends FormControlValues>({
         placeholderStyle={[{ color: colors.placeholder }, globalStyles.fs14]}
         style={[
           styles.dropDown,
-          { backgroundColor: colors.backgroundElements },
+          { backgroundColor: backgroundColor || colors.backgroundElements },
         ]}
+        dropDownContainerStyle={[
+          { backgroundColor: backgroundColor || colors.backgroundElements },
+        ]}
+        dropDownDirection={dropDownDirection || 'AUTO'}
         zIndex={zIndex}
         disabled={disabled}
         placeholder={placeholder}
+        onChangeValue={
+          onChange ? (value) => onChange(value as string) : undefined
+        }
       />
```

Comment:
May be it would be better to put it in the handleChange and check for the presence of a value and not use "as string"

Reply:
handler added